src/lib/stellar/sep1.js

```diff
@@ -1,4 +1,4 @@
-import { StellarTomlResolver } from 'stellar-sdk'
+import StellarTomlResolver from '@stellar/stellar-sdk'
```

Comment:
I think this actually should be `import { StellarToml } from '@stellar/stellar-sdk'`, and then in the `fetchStellarToml()` function, we'll then use `StellarToml.Resolver.resolve(domain)`